draft/security.md

```diff
@@ -0,0 +1,109 @@
+# Security Working Group
+
+🚧 Draft charter: this working group isn't currently approved. The proposal is for establishing a formal security team to coordinate Django's security efforts.
+
+This charter proposes the formation of an official Security Working Group to formalize and expand the security coordination efforts that have been part of Django's development process. The Security Working Group will be responsible for ensuring that Django and associated projects maintained by the Django Software Foundation maintain high security standards and respond effectively to security incidents.
+
+## Responsibilities
+
+This working group will focus on security aspects of the Django project. The team will be responsible for addressing security concerns, reviewing vulnerability reports, and providing guidance on security best practices throughout the Django ecosystem.
+
+The duties of the working group are:
+- Receive, triage, and respond to security vulnerability reports for Django
+- Coordinate the release of security patches with the Django release team
+- Review security-related pull requests across Django projects
+- Provide security guidance on Django features and APIs
+- Monitor the security landscape to proactively identify potential vulnerabilities
+- Help maintain Django's security documentation
+- Develop and maintain security-related packages in the Django ecosystem
+- Provide security advice to Django package maintainers when requested
+- Assist with security aspects of the djangoproject.com website
+- Mentor new contributors interested in Django security
+- Run or support security-focused sessions in DjangoCon sprints
+- Chair, Co-Chair and Board Liaison can sign off on security decisions
+
+## Initial membership
+
+- Chair: TBD
+- Co-Chair: TBD
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): TBD
+- Other members:
+  - TBD
+  - ...
+
+## Future membership
+
+### Who is eligible to join? Any volunteer, or are there specific requirements?
+
+Members must have interest in Django security and should have experience with web security concepts, particularly as they relate to Django applications. Prior experience with Django and its security mechanisms (e.g., CSRF protection, XSS protection) is highly recommended. Members should be familiar with the Django [security release process](https://docs.djangoproject.com/en/dev/internals/security/) or willing to learn.
+
+We welcome contributors with various levels of experience, but due to the sensitive nature of security work, members should demonstrate a track record of responsible security practices. Experience with security vulnerability assessment, analysis, or remediation is a plus.
```

Comment:
I think this needs to be a little stricter. Because the WG deals with vulnerability reports, I think this needs to retain the "invite only" nature.

Membership takes more than just "experience with web security concepts".